src/MiniEstablishments/Market/marketEventData.ts

```diff
@@ -0,0 +1,201 @@
+      }
+    },
+    chase: {
+      function (town) {
+        const npc1 = createNPC(town, {
+          profession: 'merchant'
+        })
+        return `${profile(npc1, lib.articles.output(npc1.descriptor).toUpperFirst())} is waving ${npc1.hisher} fist in the air while running after a scrawny looking child. The merchant is yelling for someone to stop that theif.`
+      }
+    },
+    animalEscape: {
+      exclusions (town) {
+        return !town.bans.includes('animals')
+      },
+      function () {
+        const animal = ['bull', 'ox', 'lion', 'wolf', 'large boar', 'hippo', 'tiger', 'elephant', 'rhino', 'leapard', 'jaguar', 'hyena', 'grizzly bear', 'crocodile', 'alligator', 'killer bunny', 'giant scorpion', 'cougar', 'gorilla']
+        const chase = ['single child', 'group of armed thugs', 'few town guards', 'angry merchant', 'distraught merchant', 'party of adventurers', 'group of mercenaries', 'few shabby looking kids']
+        return `People around start to yell and shout as they realize ${lib.articles.output(lib.random(animal))} has escaped from its cage. The animal is going wild through the market as ${lib.articles.output(lib.random(chase))} chases after it.`
+      }
+    },
+    brawl: {
+      function (town) {
+        const npc1 = createNPC(town)
+        const npc2 = createNPC(town)
+        return `A crowd has gathered near the center of the market to watch a brawl that has broken out. ${profile(npc1, lib.articles.output(npc1.descriptor).toUpperFirst())} is in the center of the crowd currently holding ${profile(npc2, lib.articles.output(npc2.descriptor))} in a headlock and shouting something that can't be made out.`
+      }
+    },
+    music: {
+      function (town) {
+        const npc = createNPC(town, {
+          hasClass: true,
+          dndClass: 'bard'
+        })
+        const instrument = ['lute', 'pair of drumes', 'harp', 'flute', 'pan flute', 'sitar', 'fiddle', 'citern', 'vielle', 'clavichord', 'harpsichord']
+        const playing = ['rather well', 'rather poorly', 'very loudly', 'quite softly', 'and it sounds pretty good', 'and it sounds bad', 'and it sounds alright']
+        return `${profile(npc, lib.articles.output('bard').toUpperFirst())} is standing in a corner of the market playing ${lib.articles.output(lib.random(instrument))} ${lib.random(playing)}. Every once in a while a passerby throws a coin into a cap sitting in front of the bard.`
+      }
+    },
+    snakeCharmer: {
+      exclusions (town) {
+        return !town.bans.includes('animals')
+      },
+      function (town) {
+        const npc = createNPC(town, {
+          profession: 'snake charmer'
```

Comment:
I suspect that this will output an error since the `snake charmer` isn't a valid profession. I'd suggest changing to a more generic 'performer' or something.